helper/m4/configure.m4

```diff
@@ -17,5 +17,18 @@ AC_ARG_ENABLE([helper-linux],
     [helper_linux=$enableval],
     [helper_linux=yes])
 
+#########################################################################
+# If libcli is available, enable CLI helper
+#########################################################################
+helper_cli=no
+LIBCLI_LIBS=""
+AC_CHECK_HEADER(libcli.h,
+    [AC_CHECK_LIB(cli, cli_init,
+                  [helper_cli=yes; LIBCLI_LIBS="-lcli"],
+                  [])],
+    [])
```

Comment:
For cross-compilation it would be useful to have a configure option for passing a custom libcli install path (e.g. `--with-libcli-path`).